consensus/src/consensus.rs

```diff
@@ -503,6 +524,48 @@ impl<R: ConsensusRpc> ConsensusStateManager<R> {
         Ok(())
     }
 
+    fn bootstrap_offline(&mut self, checkpoint: &[u8], bootstrap: &mut Bootstrap) -> Result<()> {
+        let is_valid = self.is_valid_checkpoint(bootstrap.header.slot.into());
+
+        if !is_valid {
```

Comment:
```suggestion
    if !self.is_valid_checkpoint(bootstrap.header.slot.into()) {
```
There's no need to create the `is_valid` variable just for the condition. I think it would be clearer to use its value directly.